src/rgw/motr/gc/gc.cc

```diff
@@ -87,6 +95,23 @@ bool MotrGC::going_down() {
   return down_flag;
 }
 
+int MotrGC::dequeue(const DoutPrefixProvider* dpp, std::string iname, motr_gc_obj_info obj)
+{
+  int rc;
+  bufferlist bl;
+  rc = static_cast<rgw::sal::MotrStore*>(store)->do_idx_op_by_name(iname,
+                                M0_IC_DEL, std::to_string(obj.time), bl);
+  if (rc < 0 && rc != -EEXIST){
+    ldout(cct, 0) << "ERROR: failed to delete time entry "<<obj.time<<" rc: " << rc << dendl;
+  }
+  rc = static_cast<rgw::sal::MotrStore*>(store)->do_idx_op_by_name(iname,
```

Comment:
Deleting the tag first will help in guarding against RGW crashes in any way ? If so do this first & then delete the time entry.

Reply:
Done